src/main/kotlin/org/jitsi/nlj/util/Bandwidth.kt

```diff
@@ -64,7 +65,7 @@ class Bandwidth(bps: Double) : Comparable<Bandwidth> {
     operator fun div(other: Bandwidth): Double =
         bps / other.bps
 
-    override fun compareTo(other: Bandwidth): Int = (bps - other.bps).toInt()
+    override fun compareTo(other: Bandwidth): Int = sign(bps - other.bps).toInt()
```

Comment:
Interesting, what did it warn about here?

Reply:
Nothing right here, but I ended up with unit test failures after fixing another warning. The problem is that e.g `0.5.toInt()` is `0` (and `bps` is of type `Double`)